main.js

```diff
@@ -14,11 +14,38 @@ var status = ['Applying', 'Under Interview', 'Exam Pending', 'Admitted', 'Probat
 async function acceptStudent(id, fullName, age, address){
     let attributes = { ID:id, Name: fullName, Age: age, Address: address };
     await db.put(id, attributes);
-    return await db.get(id, function(err, value){
+    await db.get(id, function(err, value){
         if(err){
             console.log(err);
         } else{
             console.log(value, status[0]);
         }
     })
+    var scheduleDate = new Date('March 1, 2021 08:30:00');
+    return await scheduleInterview(id, scheduleDate);
+}
+
+async function scheduleInterview(id, scheduleDate){
+    await db.get(id, function(err, value){
+        if(err){
+            console.log(err);
+        } else{
+            console.log(value, status[1]);
+            console.log('Interview date on', scheduleDate);
+        }
+    })
+    return await scheduleExam(id, scheduleDate);
 }
+
+async function scheduleExam(id, scheduleDate){
+    var examDate = new Date(scheduleDate);
+    examDate.setDate(examDate.getDate() + 10);
+    return await db.get(id, function(err, value){
+        if(err){
+            console.log(err);
+        } else{
+            console.log(value, status[2]);
+            console.log('Exam date on', examDate);
+        }
```

Comment:
Simply get the student, make some update, save.
```suggestion
    var student = await db.get(id);
    student.examDate = scheduleDate;
    await db.put(id, student);
```

Reply:
done sir. We did it similarly with the scheduleInterview function.